elb/src/test/java/brooklyn/entity/proxy/aws/ElbControllerLiveTest.java

```diff
@@ -107,16 +136,17 @@ public void setUp() throws Exception {
 
         // Also removes scriptHeader (e.g. if doing `. ~/.bashrc` and `. ~/.profile`, then that can cause "stdin: is not a tty")
         brooklynProperties.remove("brooklyn.ssh.config.scriptHeader");
-        
+
         mgmt = new LocalManagementContext(brooklynProperties);
-        
+
         super.setUp();
-        
+        identity = brooklynProperties.getFirst("brooklyn.location.jclouds.aws-ec2.identity");
+        credential = brooklynProperties.getFirst("brooklyn.location.jclouds.aws-ec2.credential");
```

Comment:
Throw an exception if either `identity` or `credential` is null so the reason for the test failure is clear.